src/Global/molecules/UserMenu.tsx

```diff
@@ -101,11 +102,54 @@ const LanguageMenu = styled.button`
     &:active { ${theme.typography.global.mainMenu.subItem.active}; }
   `};
 
-  padding: 20px 10px 15px;
+  border-bottom: ${({ theme: { colors } }) => colors.divider} 1px solid;
+  ${({ theme }) => css`
+    ${theme.typography.global.mainMenu.subItem.default};
+    &:hover { ${theme.typography.global.mainMenu.subItem.hover}; }
+    &:active { ${theme.typography.global.mainMenu.subItem.active}; }
+  `};
+
+  padding: 20px 10px 20px;
   align-items: center;
   width: 100%;
 `;
 
+const FooterContainer = styled.div`
+  font-family: ${({ theme }) => theme.fontFamily.ui};
+  margin-top: auto;
+  display: flex;
+  flex-direction: row;
+  padding: 10px 10px 10px 27px;
+  align-items: center;
+  width: 100%;
+  font-size: 12px;
+  font-weight: 800;
+  display: flex;
+  gap: 16px;
+`;
+
+const NavigationContainer = styled.div`
+  max-height: 300px;
+  overflow: scroll;
+  overflow-x: hidden;
+  overflow-y: auto;
+`;
+
+const FooterConatiner = styled.div`
+  white-space: break-spaces;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  user-select: none;
+  white-space: nowrap;
+`;
```

Comment:
Your naming here needs to be reviewed. It has a typo and if you correct it, it will conflict with the already defined `FooterContainer`.

Perhaps `FooterText` or `FooterMeta` might be better.

Reply:
Done